tests/Xamarin.Forms-Performance-Integration/Droid/Xamarin.Forms.Performance.Integration.Droid.csproj

```diff
@@ -79,6 +84,33 @@
     <AndroidResource Include="Resources\layout\Toolbar.axml" />
     <AndroidResource Include="Resources\values\styles.xml" />
   </ItemGroup>
+  <ItemGroup>
+    <AndroidNativeLibrary Include="$(_ASANScript)" Condition=" '$(UseASAN)' != '' ">
+        <Link>lib\arm64-v8a\wrap.sh</Link>
```

Comment:
This should be indented only 2 spaces from `<AndroidNativeLibrary/>`, not 4.  4 is for attributes, not elements.